node/node.go

```diff
@@ -471,6 +471,10 @@ func NewNode(config *cfg.Config,
 	sw.AddReactor("EVIDENCE", evidenceReactor)
 	sw.SetNodeInfo(nodeInfo)
 	sw.SetNodeKey(nodeKey)
+	err = sw.AddPersistentPeers(splitAndTrimEmpty(config.P2P.PersistentPeers, ",", " "))
+	if err != nil {
+		return nil, errors.Wrap(err, "error in persistent_peers")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "could not add peers from persistent_peers config")
```
`err` should contain what was wrong (e.g. seperator not found etc)